src/interface/metadata.hpp

```diff
@@ -54,10 +48,44 @@
+  /**  Provided by a package */                                                          \
+  PARTHENON_INTERNAL_FOR_FLAG(Provides)                                                  \
+  /**  Not created by a package, assumes available from another package */               \
+  PARTHENON_INTERNAL_FOR_FLAG(Requires)                                                  \
+  /**  does nothing if another package provides the variable */                          \
+  PARTHENON_INTERNAL_FOR_FLAG(Overridable)                                               \
+  /************************************************/                                     \
+  /**  SHAPE: Neither or one (but not both) can be specified */                          \
+  /**  a vector quantity, i.e. a rank 1 contravariant tensor */                          \
   PARTHENON_INTERNAL_FOR_FLAG(Vector)                                                    \
   /**  a rank-2 tensor */                                                                \
   PARTHENON_INTERNAL_FOR_FLAG(Tensor)                                                    \
+  /************************************************/                                     \
+  /**  DATATYPE: Exactly one must be specified (default is Real) */                      \
+  /** Boolean-valued quantity */                                                         \
+  PARTHENON_INTERNAL_FOR_FLAG(Boolean)                                                   \
+  /** Integer-valued quantity */                                                         \
+  PARTHENON_INTERNAL_FOR_FLAG(Integer)                                                   \
+  /** Real-valued quantity */                                                            \
+  PARTHENON_INTERNAL_FOR_FLAG(Real)                                                      \
+  /************************************************/                                     \
+  /**  INDEPENDENT: Exactly one must be specified (default is Independent) */            \
+  /** is an independent, evolved variable */                                             \
+  PARTHENON_INTERNAL_FOR_FLAG(Independent)                                               \
+  /** is a derived quantity (ignored) */                                                 \
+  PARTHENON_INTERNAL_FOR_FLAG(Derived)                                                   \
+  /************************************************/                                     \
+  /**  COMMUNICATION: SharedComms can only be set if FillGhost is set */                 \
+  /** Do boundary communication */                                                       \
+  PARTHENON_INTERNAL_FOR_FLAG(FillGhost)                                                 \
+  /** Communication arrays are a copy: hint to destructor */                             \
+  PARTHENON_INTERNAL_FOR_FLAG(SharedComms)                                               \
```

Comment:
I wouldn't worry about this now, but are we actually using this? With comms all reference counted, I'm not sure we are

Reply:
You're right, it doesn't seem to be used. I'm happy to remove it unless there are plans to use it in the future

Reply:
I think removing it is better. :+1: :+1: :+1: